### PR TITLE
Land ERA5 forcing data artifact

### PR DESCRIPTION
**Pull request by kmdeck**
## Purpose 
Adds the land forcing data (1 year from ERA5 at 1 degree resolution).

## To-do
update README

## Content
- create_artifacts.jl
- output artifacts.toml information
- File was transferred to cluster and Overrides.toml updated.

Review checklist

I have:
- followed the codebase contribution guide: https://clima.github.io/ClimateMachine.jl/latest/Contributing/
- followed the style guide: https://clima.github.io/ClimateMachine.jl/latest/DevDocs/CodeStyle/
- followed the documentation policy: https://github.com/CliMA/policies/wiki/Documentation-Policy
- checked that this PR does not duplicate an open PR.

In the Content, I have included 
- relevant unit tests, and integration tests, 
- appropriate docstrings on all functions, structs, and modules, and included relevant documentation.

----
- [X] I have read and checked the items on the review checklist.

### CODE REVIEW

File: era5_land_forcing_data2021/Project.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[deps]
+ArtifactUtils = "8b73e784-e7d8-4ea5-973d-377fed4e3bce"
+ClimaArtifactsHelper = "6ffa2572-8378-4377-82eb-ea11db28b255"
+Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
```
Comment (Sbozzolo) [inline era5_land_forcing_data2021/Project.toml:5]: Can you please add a newline here?

File: era5_land_forcing_data2021/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Clima Land forcing data  - ERA5 reanalysis data for 2021
+more details to follow.
```
Comment (Sbozzolo) [inline era5_land_forcing_data2021/README.md:2]: Can you also add a license/requested acknowledgments (if available)?

File: era5_land_forcing_data2021/create_artifact.jl [hunk 3]
```diff
@@ -0,0 +1,24 @@
+using Downloads
+
+using ClimaArtifactsHelper
+
+output_dir = "era5_land_forcing2021_artifact"
+const FILE_URL = "https://caltech.box.com/shared/static/yi4dlo9wug9a4yz2ckqfiqh26a61u55y.nc"
```
Comment (Sbozzolo) [inline era5_land_forcing_data2021/create_artifact.jl:6]: Maybe add a note saying that this is a very large file?

File: era5_land_forcing_data2021/README.md (hunk 2 above)
Comment (kmdeck) [reply era5_land_forcing_data2021/README.md]: I will double check what I have with @braghiere

File: era5_land_forcing_data2021/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+- Neutral wind at 10m, u-component (lon, lat, time, "u10n")
+- Neutral wind at 10m, v-component (lon, lat, time, "v10n")
+- Surface air pressure (lon, lat, time, "sp")
+- Dew point temperature at 2 meters (lon, lat, time, "d2m")
+- Total accumulated precipitation (lon, lat, time, "tp")
+- Accumulated snowfall (lon, lat, time, "sf")
+- Leaf area index, high vegetation (lon, lat, time, "lai_hv")
+- Leaf area index, low vegetation (lon, lat, time, "lai_lv")
+
+# Attribution
```
Comment (kmdeck) [inline era5_land_forcing_data2021/README.md:19]: @braghiere do you know if this is enough regarding citation of ERA5? I followed their instructions here:
https://confluence.ecmwf.int/display/CKB/Use+Case+2%3A+ERA5+hourly+data+on+single+levels+from+1940+to+present

Comment (braghiere) [reply era5_land_forcing_data2021/README.md:19]: A more formal academic citation used for ERA5 is:

Hersbach, H., Bell, B., Berrisford, P., Hirahara, S., Horányi, A., Muñoz-Sabater, J., Nicolas, J., Peubey, C., Radu, R., Schepers, D., Simmons, A., Soci, C., Abdalla, S., Abellan, X., Balsamo, G., Bechtold, P., Biavati, G., Bidlot, J., Bonavita, M., … Thépaut, J.-N. (2020). The ERA5 global reanalysis. Quarterly Journal of the Royal Meteorological Society, 146(730), 1999–2049. https://doi.org/10.1002/QJ.3803

**Pull request closed without merging**